pkg/cluster/context.go

```diff
@@ -42,16 +42,18 @@ import (
 
 // Context is used to create / manipulate kubernetes-in-docker clusters
 type Context struct {
-	name string
+	name             string
+	ControlPlaneMeta *ControlPlaneMeta
```

Comment:
I think I'd like to make this private with a cached "getter" later so that it works for contexts referring to an existing cluster. 

I'm happy to file a follow-up PR to work on this though. There's a ~similar pattern internally for `Node.KubernetesVersion()`

Reply:
If you have strong opinions I would vote we follow up with a second PR?

Reply:
SGTM